samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -72,8 +72,6 @@
           $("#jqueryui-accordion-metadata").accordion({ heightStyle: "content",collapsible: true,active: false });
           // This is a sub-accordion inside of the Availability and Submission Panel
           $("#jqueryui-accordion-security").accordion({ heightStyle: "content",collapsible: true,active: false });
-          // adjust the height of the iframe to accomodate the expansion from the accordion
-          $("body").height($("body").outerHeight() + 900);
```

Comment:
what happens when the accordion is expanded? Does it produce a scroll bar in the iframe, or does it cause the iframe to grow automatically?

Reply:
Now, there isn't any iframe. I answered the same question to @ottenhoff some days ago.

Reply:
Sorry, didn't see that. Ok, cool, thanks.